test/python/common.py

```diff
@@ -1,58 +1,5 @@
 from dataclasses import dataclass
+import binascii
 
-@dataclass
-class Keys:
-    private: bytes
-    public: bytes
-    signature: bytes
 
-
-PARTNER_KEYS = Keys(
-    private=bytes([
-        0x10, 0x67, 0xE5, 0xF6, 0xB3, 0x48, 0xEA, 0xC2, 0x68, 0xB6, 0x4F, 0xC9,
-        0xEB, 0x5A, 0x31, 0xA7, 0xD7, 0x9E, 0x33, 0xDF, 0xD6, 0xFE, 0xF7, 0x6E,
-        0xAB, 0x9F, 0x49, 0x9B, 0x47, 0xEE, 0xD6, 0x9D
-    ]),
-    public=bytes([
-        0x09, 0x46, 0x55, 0x4E, 0x44, 0x5F, 0x54, 0x45, 0x53, 0x54, 0x04, 0x00,
-        0xBE, 0xEF, 0x98, 0xB4, 0xB2, 0x99, 0x4F, 0x68, 0x8F, 0xD9, 0x8B, 0x56,
-        0x5E, 0x17, 0x1E, 0xC9, 0x25, 0x9C, 0x89, 0x06, 0x14, 0xD2, 0xDB, 0xA6,
-        0x10, 0x52, 0x02, 0x01, 0x8E, 0x3A, 0x76, 0x4A, 0xA6, 0x68, 0x3E, 0xA1,
-        0x16, 0xC9, 0xE0, 0xE3, 0xC8, 0x01, 0xCA, 0x5A, 0xB9, 0x72, 0xA2, 0xF8,
-        0xF7, 0x9F, 0x90, 0x6E, 0x19, 0xF7, 0x3E, 0xDB, 0x91, 0x45, 0x00, 0xEA,
-        0xD0, 0x2D, 0x70
-    ]),
-    signature=bytes([
-        0x30, 0x44, 0x02, 0x20, 0x3C, 0x1C, 0xE9, 0xEA, 0xF2, 0x1A, 0x18, 0x80,
-        0x3E, 0xA9, 0x87, 0x6F, 0xAB, 0x8F, 0x8A, 0x24, 0xB0, 0xD6, 0xE5, 0xF2,
-        0x6E, 0xF7, 0x54, 0xF2, 0xD1, 0xA0, 0xAC, 0x34, 0x24, 0xFC, 0xB8, 0xE1,
-        0x02, 0x20, 0x4F, 0x6C, 0x92, 0xCF, 0xEC, 0x8F, 0x23, 0x2C, 0x1B, 0x14,
-        0x89, 0x88, 0x2F, 0x98, 0x07, 0x0F, 0xE3, 0xE2, 0x00, 0x84, 0xD0, 0x23,
-        0xD6, 0x53, 0xE3, 0xEE, 0xF6, 0x96, 0xBC, 0xA0, 0x27, 0x78
-    ])
-)
-
-SWAP_PARTNER_KEYS = Keys(
-    private = bytes([
-        0x1D, 0x20, 0x2B, 0xC5, 0xEE, 0x1C, 0xED, 0x2F, 0xE7, 0xAF, 0x32, 0x38,
-        0x20, 0x16, 0xA1, 0x07, 0x35, 0x5D, 0xC0, 0xD2, 0x4E, 0x22, 0x73, 0x41,
-        0xF4, 0x31, 0x0A, 0x3C, 0x0C, 0x50, 0xD9, 0x3D
-    ]),
-    public=bytes([
-        0x09, 0x53, 0x57, 0x41, 0x50, 0x5F, 0x54, 0x45, 0x53, 0x54, 0x04, 0x66,
-        0x0A, 0x15, 0x03, 0x09, 0xFB, 0x52, 0xF3, 0xD4, 0x2C, 0x27, 0xAD, 0x04,
-        0xDC, 0x31, 0x99, 0xAA, 0x23, 0x37, 0xBD, 0x2A, 0x8A, 0x00, 0x2C, 0x53,
-        0x37, 0xD1, 0x78, 0x8A, 0xE3, 0x47, 0xD3, 0x33, 0x6E, 0x00, 0xEA, 0x33,
-        0xF4, 0x77, 0x8C, 0xD9, 0x1F, 0xF7, 0xD2, 0x8A, 0x89, 0x42, 0xEF, 0xD7,
-        0x73, 0x5D, 0xC3, 0xAD, 0x1B, 0x74, 0x53, 0xF1, 0xB9, 0xBD, 0x1F, 0xB4,
-        0xFE, 0x65, 0xFD
-    ]),
-    signature=bytes([
-        0x30, 0x44, 0x02, 0x20, 0x50, 0x0C, 0x0A, 0xCF, 0x95, 0x82, 0x00, 0x88,
-        0x4B, 0x23, 0xC6, 0xAE, 0x7F, 0xEE, 0xCB, 0x01, 0x36, 0x6D, 0x62, 0x48,
-        0xF4, 0xEA, 0xCB, 0x45, 0x2C, 0x60, 0x81, 0xE9, 0x08, 0x36, 0xDC, 0xCC,
-        0x02, 0x20, 0x7D, 0x78, 0xCB, 0xC9, 0x10, 0x3D, 0x03, 0xAB, 0x2B, 0x64,
-        0xE1, 0x61, 0x52, 0x4F, 0xB4, 0xF3, 0xCD, 0x9A, 0x88, 0x3F, 0x7F, 0xED,
-        0xBF, 0x92, 0xCB, 0x48, 0xBF, 0x7A, 0x10, 0x2D, 0x87, 0x30
-    ])
-)
+LEDGER_TEST_PRIVATE_KEY = binascii.unhexlify("b1ed47ef58f782e2bc4d5abe70ef66d9009c2957967017054470e0f3e10f5833")
```

Comment:
Since Python3.5, this is equivalent to 

```python
LEDGER_TEST_PRIVATE_KEY = bytes.fromhex("b1ed47ef58f782e2bc4d5abe70ef66d9009c2957967017054470e0f3e10f5833")
```
... except with a builtin method (not `import` needed)